daffodil-core/src/main/scala/org/apache/daffodil/dsom/SequenceGroup.scala

```diff
@@ -82,8 +81,7 @@ abstract class SequenceTermBase(
  * has the sequence properties. So actual sequences, group refs to them,
  * but NOT implied sequences inside choice branches.
  */
-abstract class SequenceGroupTermBase(
-  xml: Node,
+abstract class SequenceGroupTermBase(xml: Node,
```

Comment:
This whitespace change I can't do anything about. There's no setting for "leave it alone unless I touch it". 
If the setting leaves the first one alone and indents the 2nd and subsequent args, then the auto formatter brings the first arg up onto the line with the "(" if it fits. ... If I use the setting that puts it on the 2nd line, then they ALL, throughout the file, get converted to that.